osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -265,11 +265,11 @@ private void handleGeneral(string line)
                     break;
 
                 case @"Countdown":
-                    beatmap.Countdown = Enum.Parse<CountdownType>(pair.Value);
+                    beatmap.Settings.Countdown = Enum.Parse<CountdownType>(pair.Value);
```

Comment:
what makes this a setting? as per my comment, i don't think this should be in a separate class.

maybe editorsettings could exist, but this is not that.

Reply:
I believe `Settings` is the best term for these properties because they are all values that can be assigned and set by a medium such as the editor to a beatmap. The other only good term would've been `Metadata`, but that's already taken.

Reply:
Disagree.

Now there's `IBeatmap.Settings`, `BeatmapInfo.UserSettings` and then everything in between (`ControlPoints` are also "settings" but they aren't in settings --- where do you draw the line).

Strongly against adding a new usage of the raw term "Settings" anywhere liek this.

Reply:
Agree that "setting" isn't good, and agree in principle with splitting an interface for this additional stuff. The biggest issue I have though is how to name that new interface (classic...)

Like... best I can think of is probably `IGameplayBeatmapProperties` or something? And that still sucks.

Reply:
`IBeatmapProperties` is *far* better if we're going to keep the word "Properties".